core/controllers/creator_dashboard.py

```diff
@@ -287,7 +271,6 @@ def get(self):
 class ExplorationDashboardStatsHandler(base.BaseHandler):
     """Returns the most recent open feedback for an exploration."""
 
-    @base.require_fully_signed_up
     def get(self, exploration_id):
```

Comment:
ok. It was introduced in PR #2296 and used in  core/templates/dev/head/domain/dashboard/DashboardBackendApiService.js but I it's not used there anymore. Removing it.